aries-site/src/examples/components/pagination/PaginationServerTableExample.js

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
+import { Box, DataTable, Heading, Pagination, Text, Tip } from 'grommet';
+import { StatusCritical } from 'grommet-icons';
+
+const columns = [
+  {
+    property: 'name',
+    header: 'Name',
+    size: 'small',
+    primary: true,
+  },
+  {
+    property: 'rocket',
+    header: 'Rocket',
+    size: 'small',
+    render: datum => <Text key={datum.rocket.name}>{datum.rocket.name}</Text>,
+    align: 'end',
+  },
+  {
+    property: 'success',
+    header: 'Success',
+    size: 'xsmall',
+    align: 'end',
+    render: datum => {
+      if (datum.success === false) {
+        const content = (
+          <Box width={{ max: 'medium' }}>
+            {datum.failures?.map(({ reason }) => (
+              <Text key={reason}>{reason}</Text>
+            ))}
+          </Box>
+        );
+        return (
+          <Tip
+            plain
+            content={content}
+            dropProps={{
+              round: 'medium',
+              pad: 'small',
+              background: 'background-back',
+            }}
+          >
+            <Box>
+              <StatusCritical color="red" />
+            </Box>
+          </Tip>
+        );
+      }
+      return undefined;
+    },
+  },
+];
+
+export const PaginationServerTableExample = () => {
+  const [sort, setSort] = useState({ property: 'name', direction: 'asc' });
+  const [data, setData] = useState([]);
+  const limit = useState(10);
```

Comment:
useState returns an array where the first is the value and the second is a set function --> so this should be `const [limit, setLimit]`.

OR if you aren't changing the limit in this example, we should be able to just say `const limit = 10;`